contrib/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.5.1-alpine
+
+ENV BEANCOUNT_INPUT_FILE ""
+ENV FAVA_OPTIONS "-H 0.0.0.0"
+ENV FINGERPRINT "3f:d3:c5:17:23:3c:cd:f5:2d:17:76:06:93:7e:ee:97:42:21:14:aa"
+ENV BUILDDEPS "libxml2-dev libxslt-dev gcc musl-dev mercurial git nodejs make g++"
+ENV RUNDEPS "libxml2 libxslt"
+
+RUN cd /root \
+        && apk add --update $BUILDDEPS $RUNDEPS \
+        && hg clone --config hostfingerprints.bitbucket.org=$FINGERPRINT https://bitbucket.org/blais/beancount \
+        && (cd beancount && hg log -l1) \
+        && python3 -mpip install ./beancount \
+        && git clone https://github.com/aumayr/fava.git \
+        && (cd fava && git log -1) \
+        && make -C fava build-js \
```

Comment:
I was having the same issue here when trying to update yegle/fava docker image. I'm familiar with Python but not very much with Docker, so I'm not sure what happened here.

A quick&dirty solution would be delete the CHANGES file before `python3 -mpip install ./fava`, but I'm curious to know why it fail here.